environment.yml

```diff
@@ -2,5 +2,5 @@ channels:
     - conda-forge
 dependencies:
     - iris>=2
-    - python-eccodes>=0.9.1,<2
+    - python-eccodes
```

Comment:
Does this mean they have fixed the version-going-backwards problem ??? :stuck_out_tongue: